### ISSUE TITLE
Problem: blockchain regression fails on listed blocks with error "missing field `removed`"

### ISSUE THREAD

**Issue by sjmackenzie** [ETCDEVTeam/sputnikvm#188]
Problem: blocks
200016
200076
200219
200232

fail with this error: 
```
block 200016, transaction count: 1

working on transaction 0xa4e0ac76fe8f79ababe3c2673e59932bf707b9a2b381f376570e9241057fa392
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ErrorImpl { code: Message("missing field `removed`"), line: 1, column: 909 }', /checkout/src/libcore/result.rs:859
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::_print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic
   6: std::panicking::begin_panic_fmt
   7: rust_begin_unwind
   8: core::panicking::panic_fmt
   9: core::result::unwrap_failed
             at /checkout/src/libcore/macros.rs:29
  10: <core::result::Result<T, E>>::unwrap
             at /checkout/src/libcore/result.rs:737
  11: gethrpc::GethRPCClient::rpc_object_request
             at /home/stewart/dev/fractalide/etcdev/sputnikvm/gethrpc/src/lib.rs:162
  12: gethrpc::GethRPCClient::rpc_request
             at /home/stewart/dev/fractalide/etcdev/sputnikvm/gethrpc/src/lib.rs:142
  13: gethrpc::GethRPCClient::get_transaction_receipt
             at /home/stewart/dev/fractalide/etcdev/sputnikvm/gethrpc/src/lib.rs:304
  14: regtests::test_block
             at ./src/bin/main.rs:146
  15: regtests::main
             at ./src/bin/main.rs:259
  16: std::panicking::try::do_call
  17: __rust_maybe_catch_panic
  18: std::rt::lang_start
  19: main
  20: __libc_start_main
  21: _start
             at ../sysdeps/x86_64/start.S:120
```

**Comment by sorpaas** [comment]
@sjmackenzie This should have been fixed. Have a look.

**Comment by sjmackenzie** [comment]
still testing #191 range

**Comment by sjmackenzie** [comment]
fixed in #196
